client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -514,11 +513,11 @@ class SiteSettingsFormDiscussion extends Component {
 						<div>
 							<QueryJetpackModules siteId={ siteId } />
 
-							{ this.renderSectionHeader( translate( 'Subscriptions' ), isSubscriptionsModuleActive !== false ) }
+							{ this.renderSectionHeader( translate( 'Subscriptions' ), false ) }
 
 							<Subscriptions
 								onSubmitForm={ handleSubmitForm }
-								handleToggle={ handleToggle }
+								handleToggle={ handleAutosavingToggle }
```

Comment:
Let's also pass this prop as `handleAutosavingToggle` to the `Subscriptions` component to avoid confusion.